README.md

```diff
@@ -111,43 +111,6 @@ vt.assert_polygons(polygons_expected=polygons)
 Caveats: This repo likely misses edge cases of the many ways matplotlib plots can be created.
 Please feel free to submit bugs!
 
-
-## Active Contributors
```

Comment:
@nkorinek my apologies for not being clear. i want to keep the contributors in the readme so take what you did above and move it down here. i wanted to switch my name to the github image and then add your name below it with your github image as well.